locale/ru/docs/guides/simple-profiling.md

```diff
@@ -36,7 +36,7 @@ app.get('/newUser', (req, res) => {
 
   username = username.replace(/[!@#$%^&*]/g, '');
 
-  if (!username || !password || users.username) {
+  if (!username || !password || users[username]) {
```

Comment:
If it's a typo in original document, you need to create PR with fix, and then translate it in that way

Reply:
@alexandrtovmach It's actually already fixed there: https://github.com/nodejs/nodejs.org/commit/435c45612a8d6ae0f07d0dc18670687998e85f72#diff-d717fbc920c1977110094bda1eebfc4b

Reply:
okay, I see